docs/index.asciidoc

```diff
@@ -114,15 +114,15 @@ output plugins.
 
 [id="plugins-{type}s-{plugin}-batch_size"]
 ===== `batch_size`
-
+new[4.0.0]
```

Comment:
@karenzone Would you mind pulling these changes and test them in a build to verify the output?